src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -685,16 +685,18 @@ export type EventOrGroupType =
   | 'default'
   | 'transaction';
 
+export type InboxReasonDetails = {
+  until?: string | null;
+  count?: number | null;
+  window?: number | null;
+  user_count?: number | null;
+  user_window?: number | null;
+};
+
 export type InboxDetails = {
   date_added?: string;
   reason?: number;
-  reason_details?: {
-    until?: string;
-    count?: number;
-    window?: number;
-    user_count?: number;
-    user_window?: number;
-  };
+  reason_details: InboxReasonDetails;
```

Comment:
should we move this prop to the top of the object and leave the optional props in the end?

Reply:
Sure, fine with me!